spec/requests/profile_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'spec_helper'
+
+describe "Profile", type: :request do
+  subject { page }
+
+  describe "show" do
+    let(:user) { FactoryBot.create(:user, login: "Zaphod") }
+
+    before do
+      FactoryBot.create(:pseud, user: user, name: "Slartibartfast")
+      FactoryBot.create(:pseud, user: user, name: "Agrajag")
+      FactoryBot.create(:pseud, user: user, name: "Betelgeuse")
+      allow(ArchiveConfig).to receive(:ITEMS_PER_PAGE).and_return(3)
+    end
+
+    it "shows only the configured maximum number of pseuds" do
+      visit "/users/Zaphod/profile"
```

Comment:
We don't usually test views in specs... idk if that's something we want to start doing here?

Reply:
I definitely wanted to test the view in this case, since it didn't seem to be covered enough by unit tests, and this was the easiest way to have an integration-like test that wouldn't make me want to yeet the laptop into the sun. (The way anything cucumber does. 😭 )

Reply:
Would you like me to write the cukes, then?

Reply:
https://github.com/CristinaRO/otwarchive/pull/1 has an equivalent, the only thing it doesn't check is that it's within dl.meta.